[TW-1367] Maintenance review cycle 9/23 16-20 (#5112)
postmanlabs/postman-docs: src/pages/docs/getting-started/first-steps/creating-your-first-workspace.md

1. Select **Workspaces** in the upper left. Select **Create Workspace** to add a new workspace. You can also search for existing workspaces using the search bar or select one from the dropdown list.

    <img alt="Change workspace or create new" src="https://assets.postman.com/postman-docs/v10/workspace-switcher-v10.1.jpg" width="450px"/>

> 2. You can use [workspace templates](/docs/collaborating-in-postman/using-workspaces/creating-workspaces/#creating-workspaces-with-a-template) to help you set up a new workspace. Select a workspace template to populate the workspace with helpful information and sample collections, or select **Blank workspace** to create an empty workspace. Then select **Next**.

Reviewer: ```suggestion
1. You can use [workspace templates](/docs/collaborating-in-postman/using-workspaces/creating-workspaces/#creating-workspaces-with-a-template) to help you set up a new workspace. Select a workspace template to populate the workspace with helpful information and sample collections, or select **Blank workspace** to create an empty workspace. Then select **Next**.
```

Reviewer: We only use `1.` for numbered lists so we don't run the risk of breaking the numbering order. This way we if we add a new step we don't have to renumber list items.

Author: Thanks for catching that; the editor switched it on me and I didn't catch that.

    > You can also apply a workspace template to a newly created workspace. To learn more, see [Creating workspaces with a template](/docs/collaborating-in-postman/using-workspaces/creating-workspaces/#creating-workspaces-with-a-template).

    <img alt="Create new workspace" src="https://assets.postman.com/postman-docs/v10/create-workspace-and-apply-template-full-screen-v10-2.jpg"/>

> 3. Enter a **Name** and optional **Summary** for your workspace.
> 4. Choose who can access the workspace:

Reviewer: ```suggestion
1. Enter a **Name** and optional **Summary** for your workspace.
1. Choose who can access the workspace:
```

    * A **Personal** workspace is visible to you. If you're on an Enterprise team, [Super Admins](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) will also have access to any personal workspaces created within the team.
    * A **Private** workspace is visible to you and to any team members you invite to it ([Professional and Enterprise plans only](https://www.postman.com/pricing)).
    * A **Team** workspace is visible to everyone on your team.
    * A **Public** workspace is visible to all Postman users.

>     * A **Partner** workspace is visible to your team and your external partners. To learn more, see [Collaborate with external partners in Postman using Partner Workspaces](/docs/collaborating-in-postman/using-workspaces/partner-workspaces/). ([Postman Enterprise Ultimate plans only](https://www.postman.com/pricing))

Author: I agree! I wasn't too fond of the way it looked either.

>     * A **Partner** workspace is visible to your team and your external partners. To learn more, see [Collaborate with external partners in Postman using Partner Workspaces](/docs/collaborating-in-postman/using-workspaces/partner-workspaces/). ([Postman Enterprise Ultimate plans only](https://www.postman.com/pricing))

Author: Updated in 39ecdccd903e24f36ac6241e0ddbf9e5b85be6c2

> 5. Select **Create**.

Reviewer: ```suggestion
1. Select **Create**.
```
